### ISSUE TITLE
[Feature requests] formatted output compatible with maxbin2, metabat2, and concoct

### ISSUE THREAD

**Issue by jolespin** [bluenote-1577/fairy#2]
I'm interested in swapping out CoverM in VEBAs prokaryotic and eukaryotic binning modules.  They currently use CoverM for coverage calculations but having a faster more memory efficient alternative would be awesome. 

Is it within scope to provide utility to output (or post hoc) convert to formats compatible with different binners?

**Comment by bluenote-1577** [comment]
Hi Josh, two things:

1. Maxbin2 and MetaBAT compatible formats are available for output via several options. Metabat2 is the default option, but maxbin can be toggled.

2. Concoct is a bit weird since internally it requires some chunking of contigs, so I didn't investigate further... it seems modern binners outperform concoct as well, so I decided not to Deal with it 

3. We showed in our preprint (accepted for pub) that fairy is comparable for multiple sample binning, but for single sample coverage, it isn't as good as e.g. BWA. So I would recommend users be able to use coverm for single sample still :)
